packages/cli/src/errors/queued-execution-retry.error.ts

```diff
@@ -1,9 +1,7 @@
-import { ApplicationError } from 'n8n-workflow';
+import { UnexpectedError } from 'n8n-workflow';
 
-export class QueuedExecutionRetryError extends ApplicationError {
+export class QueuedExecutionRetryError extends UnexpectedError {
```

Comment:
Sounds like an `OperationalError` based on the description. Or `UserError` as the execution id is coming from user